Assets/Scripts/PlayerMovement.cs

```diff
@@ -41,5 +53,20 @@ void Update()
             moveVector += Physics.gravity * Time.deltaTime;
 
         cc.Move(moveVector * Time.deltaTime);
+
+        // move camera towards target (crouched or standing) height, with an exponential ease
+        float cameraNewHeight = Mathf.Lerp(cam.transform.localPosition.y, cameraTargetHeight, crouchSpeed * Time.deltaTime);
```

Comment:
Having the smooth camera is a very nice touch, and using Lerp for it is a very clean way of doing it.